examples/pxScene2d/src/pxTextBox.cpp

```diff
@@ -111,8 +117,7 @@ void pxTextBox::onInit()
   // If this is using the default font, we would not get a callback
   if(mFontLoaded || (getFontResource() != NULL && getFontResource()->isFontLoaded()))
   {
-    mFontLoaded = true;
-    setNeedsRecalc(true);
+    resourceReady("resolve");
```

Comment:
The textBox promise cannot be resolved here.  Measurements must be calculated and only then can the promise be resolved.